build.sbt

```diff
@@ -92,7 +92,7 @@ lazy val dseTest = project
   .settings(libraryDependencies ++= Dependencies.dseTestDependencies)
 
 lazy val docs = project
-  .enablePlugins(Common, PekkoParadoxPlugin, ParadoxSitePlugin, PreprocessPlugin, PublishRsyncPlugin)
```

Comment:
Why is `Common` being removed here?

Reply:
Testing if it's actually used since other projects don't have it.

Reply:
Is it okay if we do this in another PR?